SUPPORT.md

```diff
@@ -33,19 +36,19 @@ You can check your sync status in the integration settings:
 | Status   | Definition                 |
 |----------|----------------------------|
 | PENDING  | The sync has not started.  |
-| IN PROGRESS   | The sync has started and is still in progress. No information will be displayed in Jira. |
-| COMPLETE | The sync has finished. Information will be displayed in Jira. |
-| FAILED   | The sync hit an error and stopped without completing. Partial information may appear in Jira. |
+| IN PROGRESS   | The sync has started and is still in progress. No historical information will be displayed in Jira, but the integration will work for new data sent in. |
```

Comment:
That's not necessarily true. Especially if a sync takes a long time, *some* historical data will be in Jira already, even if it's still `IN PROGRESS`.

Maybe write something like: "The backfilling of historical data is in progress. Over time, more and more data will show up in Jira."